### PR TITLE
Feature/ADS-907: Add rest of package setup dependencies.

### PR DESCRIPTION
**Pull request by misdreavus79**
Added React, Rollup, Storybook, and relevant default configs for each.

### CODE REVIEW

File: src/stories/assets/discord.svg
Comment (kaseyllabelle) [inline src/stories/assets/discord.svg:1]: do we need to keep all the assets that are added here?

File: .storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { StorybookConfig } from '@storybook/react-vite';
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  addons: [
+    '@storybook/addon-onboarding',
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@chromatic-com/storybook',
```
Comment (kaseyllabelle) [inline .storybook/main.ts:9]: we don't have this in the anatomy-react storybook/main.ts file, is it something we should add?

**Review by kaseyllabelle — APPROVED** [review]

File: src/stories/assets/discord.svg
Comment (misdreavus79) [reply src/stories/assets/discord.svg:1]: No, we can clean those up. Storybook installs all of these when you first add the project. I'll take care of them.

File: .storybook/main.ts (hunk 1 above)
Comment (misdreavus79) [reply .storybook/main.ts:9]: Chromatic, you mean? It gets installed automatically with newer versions of storybook I believe, but this package won't need it. I'll clean this up as well when I clean up the assets folder.

**Review by cblva — APPROVED** [review]

**Pull request merged**